test/system/VectorListTest.ooc

```diff
@@ -227,21 +213,10 @@ VectorListTest: class extends Fixture {
 			expect(sum, is equal to(6))
 			list free()
 		})
-		this add("Iterator leak", func {
-			list := VectorList<Int> new()
-			list add(1)
-			list add(2)
-			list add(4)
-			// Convenient, but leaks the iterator instance.
-			for ((index, item) in list)
-				expect(item, is equal to(list[index]))
-			list free()
-		})
```

Comment:
Having a test that is made to leak is bad. I moved a comment about it to the next test.